src/v2/Components/UserSettings/UserInformation/__tests__/UserInformation.jest.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react"
+import { graphql } from "react-relay"
+import { createTestEnv } from "v2/DevTools/createTestEnv"
+import { UserInformationQueryResponse } from "v2/__generated__/UserInformationQuery.graphql"
+import { UserInformationRefetchContainer } from ".."
+import { UserInformationTestPage } from "./UserInformationTestPage"
+
+jest.unmock("react-relay")
+
+const setupTestEnv = () => {
+  return createTestEnv({
```

Comment:
This `createTestEnv` function setup looks great and don't think we should change, but as FYI wanted point out that we've been using the new relay-test-utils package [and helper](https://github.com/artsy/force/blob/0b291f005763e7c2600a5077786c9510bf655079/src/v2/DevTools/setupTestWrapper.tsx) for new relay test stuff, which will automatically mock data. Not sure how it works with mutations, however.